.github/testing-issue-template.md

```diff
@@ -6,11 +6,16 @@ labels: testing
 A new version ({{ env.version }}) of `{{ env.SNAP_NAME }}` was just pushed to the `{{ env.CHANNEL }}` channel [in the snap store](https://snapcraft.io/{{ env.SNAP_NAME }}). The following revisions are available.
 
 | CPU architecture | Revision                 |
-|------------------|--------------------------|
+| ---------------- | ------------------------ |
```

Comment:
Not sure why this line was changed. Is it better to have spaces here?

Reply:
Auto-formatter got me, have reverted.